<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPelanggan</class>
 <widget class="QWidget" name="FormPelanggan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kode_pelangganLabel">
      <property name="text">
       <string>Kode Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kode_pelangganLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nama_pelangganLabel">
      <property name="text">
       <string>Nama Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nama_pelangganLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamat_pelangganLabel">
      <property name="text">
       <string>Alamat Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="alamat_pelangganLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telp_pelangganLabel">
      <property name="text">
       <string>Telp Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="telp_pelangganLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="fax_pelangganLabel">
      <property name="text">
       <string>Fax Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="fax_pelangganLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="email_palangganLabel">
      <property name="text">
       <string>Email Palanggan</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="email_palangganLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="tglLabel">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="tglDateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SImpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablePelanggan">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
